test/lazyload-image.test.ts

```diff
@@ -149,6 +149,38 @@ describe('Lazy load image', () => {
 
             is(result, true);
         });
+
+        it('Should be vissible when only left side with no corners is in the viewport', () => {
+            const element = generateElement(-100, 500, 1200, 1200);
+            debugger;
```

Comment:
All these debug statement should be removed. I know, it was me how added them and I'm ashamed. 
(You do not have to do anything, I will clean up my mess)